admin/ops/setup_dev_bucket.py

```diff
@@ -0,0 +1,340 @@
+import boto3
+import importlib
+import os
+from pathlib import Path
+import typer
+
+PROD_RECIPES_BUCKET = "edm-recipes"
+PROD_PUBLISHING_BUCKET = "edm-publishing"
+os.environ["RECIPES_BUCKET"] = PROD_RECIPES_BUCKET
+os.environ["PUBLISHING_BUCKET"] = PROD_PUBLISHING_BUCKET
+
+from dcpy import configuration
+from dcpy.models.connectors.edm.recipes import DatasetKey
+from dcpy.models.connectors.edm.publishing import (
+    ProductKey,
+    BuildKey,
+    DraftKey,
+    PublishKey,
+)
+from dcpy.utils import s3
+from dcpy.utils.logging import logger
+from dcpy.connectors.edm import recipes, publishing
+from dcpy.lifecycle.builds import plan
+
+
+ROOT_PATH = Path(__file__).parent.parent
+BUCKET_PREFIX = "de-dev-"
+PRODUCTS = [
+    "db-cbbr",
+    "db-colp",
+    "db-cpdb",
+    "db-developments",
+    "db-facilities",
+    "db-green_fast_track",
+    "db-template",
+    "db-pluto",
+    "db-zoningtaxlots",
+]
+
+
+def copy_folder(
+    source_bucket: str,
+    target_bucket: str,
+    path: str,
+) -> None:
+    """Copies folder from a production bucket to a dev bucket"""
+    assert target_bucket.startswith(BUCKET_PREFIX)
+    logger.info(f"Copying {source_bucket}/{path} to ./tmp")
+    s3.download_folder(
+        source_bucket, prefix=path, export_path=Path("./tmp/dev_bucket_setup")
+    )
+    logger.info(f"Copying ./tmp to {target_bucket}/{path}")
+    s3.upload_folder(
+        target_bucket,
+        Path(f"./tmp/dev_bucket_setup/{path}"),
+        Path(path),
+        "private",
```

Comment:
private acl... who does this exclude/include?

Reply:
Just means that you need credentials to access - public read means anyone with url can access. Figured might as well make it private. Maybe at least I'll pull that out to a constant at the top of the script